apps/cyberstorm-storybook/stories/components/Tooltip.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Button, Tooltip } from "@thunderstore/cyberstorm";
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+
+const meta = {
+  title: "Cyberstorm/Components/Tooltip",
+  component: Tooltip,
+} as ComponentMeta<typeof Tooltip>;
+
+const buttonargs = {
+  label: "Categories",
+  rightIcon: (
+    <FontAwesomeIcon
+      fixedWidth={true}
+      icon={faChevronDown}
+      className={"buttonIcon"}
+    />
+  ),
+};
+
+const defaultArgs = {
+  label: "Categories",
+};
+
+const Template: ComponentStory<typeof Tooltip> = (args) => (
+  <div
+    style={{
+      display: "flex",
+      flexDirection: "column",
+      gap: "0.25em",
+      backgroundColor: "#7e26b6",
+      padding: "1rem",
+    }}
+  >
+    <div>
+      <Tooltip {...args}>
+        <Button {...buttonargs} />
+      </Tooltip>
+    </div>
+    <div>
+      <Tooltip {...args}>
+        <Button {...buttonargs} />
+      </Tooltip>
+    </div>
+    <div>
+      <Tooltip {...args}>
+        <Button {...buttonargs} />
+      </Tooltip>
+    </div>
+    <div>
+      <Tooltip {...args}>
+        <Button {...buttonargs} />
+      </Tooltip>
+    </div>
+  </div>
+);
+
+const DefaultTooltip = Template.bind({});
+DefaultTooltip.args = defaultArgs;
+
```

Comment:
Could probably use another exmple of what happens when basicly no props are passed. as in empty parameters.
The pattern is:
MinimalTooltip (minimal to no props passed)
ReferenceTooltip (matching the design)
ExtremeTooltip ("stupid" extreme case, for example 500 chars long label etc.)

Reply:
What I meant by Minimal was to (in this case) leave the props empty, so it acts as an example of what happens when no label is passed. Since the label is an optional prop.

Reply:
Also the long one could be even longer, like 200 chars or something :D

Reply:
Done and done. Although I'm not sure why the text doesn't get wrapped to another line with the Radix's Tooltip. It just goes off the screen. `overflow-wrap` doesn't help.

Reply:
^ This is because there aren't any constraints to enable wrapping of the content, because:

1) The tooltip is rendered through portal to body, so the parent isn't constraining the width
2) It's has fixed positioning
3) There's no max-width set for the tooltip

Reply:
Oksamies, feel free to resolve once you've read gkurck's comment

Reply:
Alright, I've added
`  max-width: 75vw;
  overflow-wrap: break-word;`
And I think that could be fine, it at least looks fine. I hope that is fine.